spec/rollbar/encoding/encoder_spec.rb

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+require 'spec_helper'
+
+unless RUBY_VERSION.start_with?('1.8')
+  require 'rollbar/encoding/encoder' unless
```

Comment:
I find the `unless` is a bit confusing.

Btw, I am wondering if you should unit test this Encoder class? What do you think about treating this class as private and test the Encoding class instead?